integration_test/tests/wallet.rs

```diff
@@ -314,17 +314,37 @@ fn wallet__import_address() {
 #[cfg(not(feature = "v20_and_below"))]
 fn wallet__import_descriptors() {
     use node::{serde_json, ImportDescriptorsRequest};
+    use std::time::{SystemTime, UNIX_EPOCH};
 
     let node = Node::with_wallet(Wallet::None, &[]);
     let wallet_name = "desc_wallet";
     node.client.create_wallet_with_descriptors(wallet_name).expect("create descriptor wallet");
```

Comment:
Perhaps feature gate and then we only need to call the macro that defines this function in v21 and v22, making the default descriptor wallet thing explicit.

```rust
    #[cfg(feature = "v22_and_below")]
    node.client.create_wallet_with_descriptors(wallet_name).expect("create descriptor wallet");

    // v23 onwards uses descriptor wallets by default.
    #[cfg(not(feature = "v22_and_below"))]
    node.client.create_wallet(wallet_name).expect("create wallet"); 
```